quadrat/sass/blocks/_navigation.scss

```diff
@@ -45,7 +45,7 @@
 						.wp-block-pages-list__item__link,
 						.wp-block-navigation-link__content {
 							padding: 0;
-							font-size: 15px;
+							font-size: 16px;
```

Comment:
This should use the variable `var(--wp--preset--font-size--tiny)`

Reply:
Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b